app/src/main/java/co/nimblehq/compose/crypto/ui/components/chartintervals/TimeIntervals.kt

```diff
@@ -0,0 +1,9 @@
+package co.nimblehq.compose.crypto.ui.components.chartintervals
+
+enum class TimeIntervals(val text: String) {
+    ONE_DAY("1D"),
+    ONE_WEEK("7D"),
+    ONE_MONTH("1M"),
+    ONE_YEAR("1Y"),
+    FIVE_YEAR("5Y")
```

Comment:
I think It would be great if we create `const val` for these 🙏

Reply:
@hoangnguyen92dn @Wadeewee we don't need to define const for enum field values.